lesson_06/task_1/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func requestInfo(w http.ResponseWriter, req *http.Request) {
+	var ins response
+
+	ins.Host = req.Host
+	v := req.Header["Accept"]
+	v1 := req.Header["User-Agent"]
+	ins.Headers.UserAgent = v1
+	ins.Headers.Accept = v
+	ins.RequestURI = req.RequestURI
+	ins.UserAgent = v1[0]
+
+	js, err := json.Marshal(ins)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	w.Write(js)
+}
+
+type requestHeaders struct {
```

Comment:
You forget about about other headers that can be passed.
The HW task did not say that only this headers were needed =)
https://en.wikipedia.org/wiki/List_of_HTTP_header_fields
Example: 
`curl "http://127.0.0.1:8081/headers" -H "Max-Forwards: 10"` should return:
```
{
  "Host": "127.0.0.1:8080",
  "UserAgent": "curl/7.68.0",
  "RequestURI": "/headers",
  "Header": {
    "Accept": [
      "*/*"
    ],
    "Max-Forwards": [
      "10"
    ],
    "User-Agent": [
      "curl/7.68.0"
    ]
  }
}
```